VocabularyTrainer/NewLanguage/NewLanguageViewController.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  NewLanguageViewController.swift
+//  VocabularyTrainer
+//
+//  Created by Mariana Brasil on 16/12/22.
+//  Copyright © 2022 mic. All rights reserved.
+//
+
+import UIKit
+
+final class NewLanguageViewController: UIViewController {
+
+    // MARK: - Public Properties
+
+    weak var delegate: NewLanguageScreenProtocol?
+    var coordinator: MainCoordinator?
+
+    // MARK: - Private Properties
+
+    private lazy var closeButton: UIButton = {
+        let button = UIButton()
+        button.backgroundColor = UIColor(named: "closeButton")
+        button.layer.cornerRadius = 3
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.accessibilityTraits = .button
+        button.accessibilityLabel = Localizable.close.localize()
+        button.addTarget(self,
+                         action: #selector(closeButtonAction),
+                         for: .touchUpInside)
+        return button
+    }()
+
+    private lazy var titleLabel: UILabel = .createLabel(font: .systemFont(ofSize: 20, weight: .semibold),
```

Comment:
## AccessibilityPolice 🚓 

When using this initializer you will have a fixed fontsize. We want dynamic type :)

When there are existing iOS fontsizes that work it can be easily replaced with e.g. `.preferredFont(forTextStyle: .body)` (here is a quick overview about all the existing ones: https://www.iosfontsizes.com/ ). If not, you can use `UIFontMetrics` for creating a dynamic one.

We should probably add some helper method/extension to `UIFont` later for quickly creating dynamic type fonts.

I just tried it out this looks good I think:
```swift
    let titleFont = UIFontMetrics(forTextStyle: .body).scaledFont(for: .systemFont(ofSize: 20, weight: .semibold))
    private lazy var titleLabel: UILabel = .createLabel(font: titleFont,
                                                        text: Localizable.addNewLanguage.localize(),
                                                        accessibilityTrait: .header)

    private lazy var languageLabel: UILabel = .createLabel(font: .preferredFont(forTextStyle: .headline),
                                                           text: Localizable.language.localize(),
                                                           accessibilityTrait: .header)
    private lazy var textField: UITextField = {
        let textField = UITextField()
        textField.translatesAutoresizingMaskIntoConstraints = false
        textField.placeholder = Localizable.whichLanguage.localize()
        textField.font = .preferredFont(forTextStyle: .body)

///...
```

For example this it how it would look at size `Accessibility XL` with dynamic type font variants:

![image](https://user-images.githubusercontent.com/35889530/208307568-02e92f43-660b-4093-9ec7-9fa25c1fee10.png)

The default scaling would still look like that:
![image](https://user-images.githubusercontent.com/35889530/208307595-fe54c962-daed-4874-80c3-4040e2aa582b.png)

Reply:
ah and I forgot, of course we should do the same for the buttons :)

As you see in my screenshot with increased font size, the button stays the same.

Reply:
<img src="https://user-images.githubusercontent.com/69827434/208325290-a70ca0a6-e534-44ad-a765-1040dbe57f85.png" width="200" height="400" /><img src="https://user-images.githubusercontent.com/69827434/208325303-a5cc8ee8-190d-4ba3-b287-e077a88637e7.png" width="200" height="400" /><img src="https://user-images.githubusercontent.com/69827434/208325321-d6f4ff73-2d4b-43c4-8e32-ef12a634f1de.png" width="200" height="400" />